### PR TITLE
Admin user

### PR DESCRIPTION
**Pull request by MuneebDevs**

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class CompaniesController < ApplicationController
   before_action :set_company, only: [:show, :update, :destroy]
-
+  before_action :authenticate_admin!;
```
Comment (muneebdevsinc) [inline app/controllers/companies_controller.rb]: Please remove the semi-colon.

File: app/models/admin.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Comment (muneebdevsinc) [inline app/models/admin.rb]: Avoid leaving comments in models, controllers and views.

File: app/controllers/companies_controller.rb (hunk 1 above)
Comment (MuneebDevs) [reply app/controllers/companies_controller.rb]: done

File: app/models/admin.rb (hunk 2 above)
Comment (MuneebDevs) [reply app/models/admin.rb]: done

**Pull request merged**